lua/lsp-file-operations.lua

```diff
@@ -108,4 +117,19 @@ M.setup = function(opts)
   end
 end
 
+--- The extra client capabilities provided by this plugin. To be merged with
+--- vim.lsp.protocol.make_client_capabilities() and sent to the LSP server.
+M.default_capabilities = function()
```

Comment:
@antosha417 I don't do much lua development - so improvements welcome. If you don't like it we can also remove it again. But that would make setup slightly more verbose.